optimize/starter/README.md

```diff
@@ -0,0 +1,24 @@
+Hey there!
+
+Welcome to Yet Another Todo App: Optimize Edition!
+
+To get started, follow these steps:
+
+1. Run `npm i` from project root.
+1. Login and create yourself an API key (also called token) [in the Optimize dashboard](https://optimize.prisma.io).
+1. Navigate to `packages/server` and add a `.env` file that matches the `.env.example` file.
```

Comment:
```suggestion
1. Run `npm i` from project root.
2. Login and create yourself an API key (also called token) [in the Optimize dashboard](https://optimize.prisma.io).
3. Navigate to `packages/server` and add a `.env` file that matches the `.env.example` file.
```

Reply:
This is just md shorthand. It changes itself to be properly numbered.

<img width="1159" alt="image" src="https://github.com/user-attachments/assets/516ffecd-4d2c-44a1-8899-821915d78f82">